cmd/clusterctl/cmd/root.go

```diff
@@ -163,7 +164,9 @@ func initConfig() {
 		}
 	}
 
-	logf.SetLogger(logf.NewLogger(logf.WithThreshold(verbosity)))
+	log := logf.NewLogger(logf.WithThreshold(verbosity))
```

Comment:
Nice one! (de-encapsulate to encapsulate it again later :))

Reply:
I just extracted the statement we had before :) (didn't really think about it)